curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646a422b36a6c506d4220c2b.md

```diff
@@ -0,0 +1,119 @@
+---
+id: 646a422b36a6c506d4220c2b
+title: Step 1
+challengeType: 0
+dashedName: step-1
+---
+
+# --description--
+
+Before starting the project, you need to set up the HTML document with some basic structure. First add the `DOCTYPE`, `html`, `head`, and `body` tags. Next up, add a `meta` tag inside the `head` tag that sets the character encoding.
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.
The first step should focus on JavaScript only.

Reply:
This answered my question

Reply:
removed